resources/eventing/charts/nats/templates/configmap.yaml

```diff
@@ -155,3 +155,123 @@ data:
     {{- with .Values.nats.limits.lameDuckDuration }}
     lame_duck_duration: {{ . }}
     {{- end }}
+
+    {{- if .Values.auth.enabled }}
+    ##################
+    #                #
+    # Authorization  #
+    #                #
+    ##################
+    {{- if .Values.auth.resolver }}
+      {{- if eq .Values.auth.resolver.type "memory" }}
+    resolver: MEMORY
+    include "accounts/{{ .Values.auth.resolver.configMap.key }}"
+      {{- end }}
+      {{- if eq .Values.auth.resolver.type "full" }}
```

Comment:
Do we need to support other resolver types (like `full`) atm ? At least for the use case of the issue the `memory` resolver is enough.

Reply:
This would reduce the amount of changes which we need to maintain a lot 🤔